src/test/java/de/tum/in/www1/artemis/TextExerciseIntegrationTest.java

```diff
@@ -129,6 +129,37 @@ public void createTextExercise() throws Exception {
         assertThat(newTextExercise.getCourseViaExerciseGroupOrCourseMember().getId()).as("exerciseGroupId was set correctly").isEqualTo(course.getId());
     }
 
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void createTextExercise_setExerciseTitleNull_badRequest() throws Exception {
+        final Course course = database.addCourseWithOneReleasedTextExercise();
+        TextExercise textExercise = textExerciseRepository.findByCourseId(course.getId()).get(0);
+
+        textExercise.setTitle(null);
+
+        request.postWithResponseBody("/api/text-exercises/", textExercise, TextExercise.class, HttpStatus.BAD_REQUEST);
+    }
+
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void createTextExercise_setAssessmentDueDateWithoutExerciseDueDate_badRequest() throws Exception {
+        final Course course = database.addCourseWithOneReleasedTextExercise();
+        TextExercise textExercise = textExerciseRepository.findByCourseId(course.getId()).get(0);
+
+        textExercise.dueDate(null);
+
+        request.postWithResponseBody("/api/text-exercises/", textExercise, TextExercise.class, HttpStatus.BAD_REQUEST);
+    }
+
+    @Test
+    @WithMockUser(value = "instructor1", roles = "TA")
+    public void createTextExercise_asTutor_forbidden() throws Exception {
```

Comment:
A tutor won't be able to enter the REST call you're testing because of the `@PreAuthorize` statement. Instead what you should do is set the instructor you're testing to not be an instructor for the given course. You can simulate that relatively easily:
The courses have groups with names to which the different roles belong. In the DatabaseUtilService they are "tumuser" for users, "tutor" for tutors, "instructor" for instructors, and "admin" for admins. So, you can just set the instructor group name of the course you created to be something other than "instructor" and save the course to the course repository.